docs/self-managed/zeebe-deployment/operations/backups.md

```diff
@@ -1,49 +1,56 @@
 ---
 id: backups
 title: "Backups"
-description: "Overview and Guide for taking Zeebe backups"
+description: "A guide to creating and installing Zeebe backups."
 ---
 
-As Zeebe fully manages the state of your process instances you might want to consider taking backups of Zeebe data.
-Zeebe is fault-tolerant and replicates state internally.
-Backups are only necessary if you want to protect against the loss of entire replica sets or data corruption bugs.
+As Zeebe fully manages the state of your process instances, you should consider taking backups of Zeebe data.
 
-State of other components, such as Operate and Tasklist, is not managed by Zeebe and needs to be backed up separately.
+[//]:# ("...taking backups of Zeebe to prevent data loss?")
```

Comment:
I agree that backups aren't strictly to protect against data loss/corruption. For example, they can be useful to rollback application level errors (i.e. a "bug" the user's deployed process, but from the Zeebe point of view it's not a bug). We could simply say backups are especially useful, or are crucial, to protect against...etc.

Reply:
Okay, how about "As Zeebe fully manages the state of your process instances, consider taking backups of Zeebe data; this is crucial to prevent data loss, roll back application-level errors, and more."

I really just want to stress the importance of the doc/backups to the reader.